ESP32LapTimer/src/Wireless.h

```diff
@@ -0,0 +1,15 @@
+#include <Arduino.h>
+#include <WiFi.h>
+#include <DNSServer.h>
+
+extern IPAddress apIP;
```

Comment:
Still two externs here. If they're not needed they should be omitted

Reply:
Doesn't look like they are used anywhere. Probably something i did when moving code around. I'll remove them. Thinking about it though that file doesn't even need the includes does it?

Reply:
Yes, no includes are needed.
But I just noticed you forgot adding the include guard ;)

Reply:
Fixed in 02a8850